adafruit_esp32spi/adafruit_esp32spi_wifimanager.py

```diff
@@ -75,7 +101,17 @@ def connect(self):
             print("MAC addr:", [hex(i) for i in self._esp.MAC_address])
             for access_pt in self._esp.scan_networks():
                 print("\t%s\t\tRSSI: %d" % (str(access_pt['ssid'], 'utf-8'), access_pt['rssi']))
-        failure_count = 0
```

Comment:
Removing this line breaks line 125 below

Reply:
Fixed in the latest commit.